tutorial.adoc

```diff
@@ -85,12 +94,15 @@ openssl genpkey -algorithm RSA -out unblu.pem -pkeyopt rsa_keygen_bits:2048
 openssl rsa -pubout -in unblu.pem -out unblu_public.pem
 ----
 
-== Generate a JSON Web Token (JWT)
+== Generating a JWT
 
-The following code samples are part of a sample implementation of Visitor SSO and can be freely copied.
+The code samples that follow are part of a sample implementation of visitor SSO and may be freely copied.
 
 The application needs to create a JWT signed with an RSA key pair and make it accessible from the browser JavaScript after the user authenticated with his application specific credentials.
-The JWT needs to be signed with a key pair referenced by its Key ID (`kid`) and include the user attributes as claims named as configured in the `claimMapping` in the Unblu configuration)
+
+The JWT must be signed with a key pair referenced by its Key ID (`kid`). 
+It must include the user attributes as claims.
+The claims must be named as configured in the http://docs.unblu.com/en/docs/latest/reference/configuration-properties.html#com-unblu-platform-server-core-auth-configuration-claimmappingconfiguration[`claimMapping`^] in the Unblu configuration.
```

Comment:
@kadang shouldn't we use `www.unblu.com` instead of `docs.unblu.com`?

Reply:
Good catch. We shouldn't use `docs,unblu.com`. I normally use bare `unblu.com`.

```suggestion
The claims must be named as configured in the http://unblu.com/en/docs/latest/reference/configuration-properties.html#com-unblu-platform-server-core-auth-configuration-claimmappingconfiguration[`claimMapping`^] in the Unblu configuration.
```

Reply:
I would use https://www.unblu.com to avoid the extra redirects.

Reply:
:+1: I'll raise an issue to that in the docs, too, in that case.